File: MyHouse/app/src/main/java/com/example/myhouse/timeconverter/TimeConverter.java
package com.example.myhouse.timeconverter;

public class TimeConverter {
    private int Hours;
    private int Minutes;

    public TimeConverter() {
    }

    public String ConvertFromMinutes(int AllMinutes) {
        Hours = AllMinutes / 60;
        Minutes = AllMinutes % 60;
        if (Hours < 10 && Minutes < 10) {
            return "0" + Hours + " : 0" + Minutes;
        } else if (Hours < 10 && Minutes >= 10) {
            return "0" + Hours + " : " + Minutes;
        } else if (Hours >= 10 && Minutes < 10) {
            return Hours + " : 0" + Minutes;
        } else
            return Hours + " : " + Minutes;
    }

    public int getHours(int AllMinutes) {
        Hours = AllMinutes / 60;
        return Hours;
    }

    public int getMinutes(int AllMinutes) {
        Minutes = AllMinutes % 60;
        return Minutes;
    }
}

File: MyRoom/app/src/main/java/com/example/myroom/timeconverter/TimeConverter.kt
package com.example.myroom.timeconverter

class TimeConverter {
    private var Hours: Int? = null
    private var Minutes: Int? = null

    fun ConvertFromMinutes(AllMinutes: Int): String? {
        Hours = AllMinutes / 60
        Minutes = AllMinutes % 60
        return if (Hours!! < 10 && Minutes!! < 10) {
            "0$Hours : 0$Minutes"
        } else if (Hours!! < 10 && Minutes!! >= 10) {
            "0$Hours : $Minutes"
        } else if (Hours!! >= 10 && Minutes!! < 10) {
            Hours.toString() + " : 0" + Minutes
        } else Hours.toString() + " : " + Minutes
    }

    fun getHours(AllMinutes: Int): Int {
        Hours = AllMinutes / 60
        return Hours as Int
    }

    fun getMinutes(AllMinutes: Int): Int {
        Minutes = AllMinutes % 60
        return Minutes as Int
    }
}